sdk/identity/azure-identity/src/main/java/com/azure/identity/ClientCertificateCredentialBuilder.java

```diff
@@ -24,7 +26,18 @@
      */
     public ClientCertificateCredentialBuilder pemCertificate(String certificatePath) {
         ValidationUtil.validateFilePath(getClass().getSimpleName(), certificatePath, "Pem Certificate Path");
-        this.clientCertificate = certificatePath;
+        this.clientCertificatePath = certificatePath;
+        return this;
+    }
+
+    /**
+     * Sets the client certificate for authenticating to AAD.
```

Comment:
Sets the input stream holding the pem certificate for... ?

Reply:
updated